papers/aleksandar_makelov/myst.yml

```
@@ -0,0 +1,44 @@
+version: 1
+project:
+  # Update this to match `scipy-2024-<folder>` the folder should be `<firstname_surname>`
+  id: scipy-2024-aleksandar_makelov
+  title: "Mandala: Compositional Memoization for Simple & Powerful Scientific Data Management"
+  subtitle: LaTeX edition
+  # Authors should have affiliations, emails and ORCIDs if available
+  authors:
+    - name: Aleksandar Makelov
+      email: aleksandar.makelov@gmail.com
+      orcid: 0009-0003-5394-112X
+      affiliations:
+        - MIT
+        - SERI MATS
+        - Independent Researcher
+  keywords:
+    - data management
+    - framework
+  # Add the abbreviations that you use in your paper here
+  abbreviations:
+    MyST: Markedly Structured Text
+  # It is possible to explicitly ignore the `doi-exists` check for certain citation keys
+  error_rules:
+    - rule: doi-exists
+      severity: ignore
+      keys:
+        - bojarski2016end
+        - yang2024swe
+        - norvig1991techniques
+        - makelov2023practical
+        - git
+        - maymounkov2018koji
+        - semver
+        - lozano2017unison
+  # A banner will be generated for you on publication, this is a placeholder
+  banner: banner.png
+  # The rest of the information shouldn't be modified
+  subject: Research Article
+  open_access: true
+  license: CC-BY-4.0
+  venue: Scipy 2024
+  date: 2024-07-10
```

Regarding the broken section references in your MyST site that render as "??" - These are `\ref` commands looking for a section number, but your sections are not numbered.

Probably the easiest fix for this is to add to your `myst.yml`
```suggestion
  date: 2024-07-10
  numbering:
    headings: true
```

---

(Numberings will also be fixed on the final scipy proceedings site, but adding the change I suggested will fix your preview build.)

---

So after all the suggested changes, none of the figures are showing for me when I preview locally. I get the following type of errors:
```
convert-im6.q16: attempt to perform an operation not allowed by the security policy `PDF' @ error/constitute.c/IsCoderAuthorized/421.
convert-im6.q16: no images defined `_build/site/public/fig1-a9a37853ebabe041933dc9811dd4708b.png' @ error/convert.c/ConvertImageCommand/3229.
```
```
⛔️ main.tex:413:5 Cannot convert image "list-46402ef1b912376ea7cea258654f06ad.pdf" - may not correctly render.
   To convert this image, you must install imagemagick
⛔️ _build/site/public/fig5-12a209743562173f048a769ce611b87d.pdf Could not convert from PDF to PNG: _build/site/public/fig5-12a209743562173f048a769ce611b87d.pdf - Error: Command failed: convert -density 600 -colorspace RGB _build/site/public/fig5-12a209743562173f048a769ce611b87d.pdf -trim _build/site/public/fig5-12a209743562173f048a769ce611b87d.png
convert-im6.q16: attempt to perform an operation not allowed by the security policy `PDF' @ error/constitute.c/IsCoderAuthorized/421.
convert-im6.q16: no images defined `_build/site/public/fig5-12a209743562173f048a769ce611b87d.png' @ error/convert.c/ConvertImageCommand/3229.
```
```
⚠️  _build/site/public/fig3-340c6c379bf85d1626441d78f8180d55.pdf Could not convert to webp:
Error: Command failed: convert -density 600 -colorspace RGB _build/site/public/fig3-340c6c379bf85d1626441d78f8180d55.pdf -trim _build/site/public/fig3-340c6c379bf85d1626441d78f8180d55.png
convert-im6.q16: attempt to perform an operation not allowed by the security policy `PDF' @ error/constitute.c/IsCoderAuthorized/421.
convert-im6.q16: no images defined `_build/site/public/fig3-340c6c379bf85d1626441d78f8180d55.png' @ error/convert.c/ConvertImageCommand/3229.
```